src/Header.tsx

```diff
@@ -2,9 +2,17 @@ import * as React from 'react'
 import headerConfig from './config/headerConfig'
 
 export const Header: React.SFC<{}> = () => {
-  const { summary, title, showBlur = true, centerText = false } = headerConfig
+  const {
+    summary,
+    title,
+    showBlur = true,
+    centerText = false,
+    HeaderSvg,
+  } = headerConfig
+  const hasImg = HeaderSvg !== undefined
   return (
     <header id="header">
+      {hasImg && <HeaderSvg />}
```

Comment:
This looks good to me. I agree that if we want to scale the image properly without cropping, adding the image directly to the markup is easier.